pages/docs/getting-started/hello-world.md

```diff
@@ -7,7 +7,7 @@ menu:
 weight: 30
 ---
 
```

Comment:
@derberg, can you confirm this is technically accurate? I am not sure myself 😬

Reply:
actually it is technically correct. AsyncAPI file should be used to describe what users/consumers can do with the file. So if the file has `publish` operation, it means others can publish to this `hello` channel, and `Hello world application` will receive the message

Reply:
ok got it, I have made adjustments which I think will bring more light to those reading the sample and be confused as I was.

Reply:
ok, but `capable of publishing` should still say `capable of receiving` as the AsyncAPI doc describes `Hello world application` app